src/main/scala/uk/gov/nationalarchives/tdr/api/db/repository/FileMetadataRepository.scala

```diff
@@ -55,6 +55,7 @@ class FileMetadataRepository(db: Database)(implicit val executionContext: Execut
     db.run(query.result)
   }
 
+  // TODO: Since we are not gonna use this method anymore after refactoring (TDR-2477), please remove this method.
```

Comment:
Can you reword this comment to: `This method should be removed as part of the refactoring for TDR-2477: https://national-archives.atlassian.net/browse/TDR-2477`